CHANGES.rst

```diff
@@ -49,6 +49,8 @@ assign_wcs
 associations
 ------------
 
+- Asn_Lv2NRSLAMPSpectral: Allow msaspec only if msametfl is available. [#6085]
```

Comment:
This needs to go up at the top under the 1.2.1 release heading

Reply:
too easy to miss those headings :-(